lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -244,7 +244,7 @@ def formatted_message_and_backtrace(colorizer)
         end
 
         def exception_backtrace
-          exception.backtrace || []
+          RSpec.configuration.backtrace_formatter.format_backtrace(exception.backtrace) || []
```

Comment:
@myronmarston 
> If you have a file of code that you don't want RSpec to consider to be part of the available source code from which to draw snippets, then configure project_source_dirs appropriately (or move the file to another directory)

Yes, this workaround will work. Thanks for the help. I'll close PR
Is there any way to include specific gem to project_source_dirs ?
We use couple of gems in out project and I'd like to see snippets from gem's code